src/python/include/Alpha_complex_factory.h

```diff
@@ -63,13 +63,13 @@ class Exact_Alphacomplex_dD : public Abstract_alpha_complex {
       alpha_complex_(boost::adaptors::transform(points, pt_cython_to_cgal<Point>)) {
   }
 
-  std::vector<double> get_point(int vh) override {
+  virtual std::vector<double> get_point(int vh) override {
```

Comment:
`virtual` is redundant here, but it doesn't hurt, so we can keep it, ok.

Reply:
Yes `virtual` is not mandatory, but most people let it.